src/libraries/ExcelConverter/ExcelParser.cs

```diff
@@ -40,14 +40,22 @@ public class ParsedTable
         {
             public string Name;
             public string Range;
+            public String SheetName;
             public List<ParsedTableColumn> Columns;
+            public UInt32 HeaderRowCount;
+            public UInt32 TotalsRowCount;
```

Comment:
I think you can just use type 'int' instead of UInt32

Reply:
will resolve later during refactoring phase